source/base/patterns.cc

```diff
@@ -755,32 +755,32 @@ namespace Patterns
   {
     if (description.compare(0, std::strlen(description_init), description_init) == 0)
       {
-        int min_elements, max_elements;
+        unsigned int min_elements, max_elements;
```

Comment:
This was already an issue (like most things I found), but should we initialize these values?

Reply:
I guess initializing doesn't hurt here.